src/background/images.ts

```diff
@@ -55,7 +55,14 @@ export async function takeScreenshot(
       targetId: tab.id,
     })
   } catch (error) {
-    dispatchBackgroundActions.imageCaptureFailure({ targetId: target.id, error })
+    console.error(error)
```

Comment:
I think the `console.error` is redundant here

Reply:
The failure doesn't include the stack trace, so useful to see that here for an error we really don't expect